jquery.dataTables/jquery.dataTables.d.ts

```diff
@@ -629,6 +634,13 @@ declare namespace DataTables {
         data(): DataTable;
 
         /**
+        * Iterate over each selected cell, with the function context set to be the cell in question. Since: DataTables 1.10.6
+        *
+        * @param fn Function to execute for every cell selected.
+        */
+        every(fn: (cellRowIdx?: number, cellColIdx?: number, tableLoop?: number, cellLoop?: number) => void): DataTable;
```

Comment:
Callback parameters should generally be non-optional; users can still define a function that takes fewer parameters than are provided if they want to.